src/actions/build/mod.rs

```diff
@@ -76,11 +71,10 @@ impl Build {
             }
         }
 
-        fn convert_build_results_to_notifications(build_results: &Results,
-                                                  project_path: &Path)
+        fn convert_results_to_notifications(results: &Results, project_path: &Path)

```

Comment:
It's not really your fault, more a weakness in Git, but this diff is really hard to review. When you're doing refactoring, could you make your commits as small as possible please? E.g., here, I think factoring out `process_build_result` should be its own commit, deleting the structs one commit, and the other renaming another commit.